breaking/entrypoint.sh

```diff
@@ -9,6 +9,9 @@ write_output () {
             file_output=$output
         fi
         echo "$file_output" >> "$output_to_file"
+    elif [ -n "$2" ]; then
+      # if we don't have a file to write to use the full content of the file output if specified
+      output="$2"
```

Comment:
We want to write to the output only summary.
Is this related to https://github.com/oasdiff/oasdiff-action/issues/46?

Reply:
this is indeed unrelated. it was my first assumption, that the output got swallowed by this somehow. i removed it